lib/utils/convertToDailyOverview.js

```diff
@@ -0,0 +1,52 @@
+const { createTime } = require("./convertTimestamp");
+
+const buildDailyOverview = (documents, { now }) => {
```

Comment:
Instead of destructing the second argument, you could directly bypass the timestamp.
And I would rename it to allow adding timestamps of other days.

```suggestion
const buildDailyOverview = (documents, timestamp) => {
```

Reply:
i corrected it in the script higher in the hierarchy, to just submit the data needed